### ISSUE TITLE
Community Committee Director Election - Call for Nominations

### ISSUE THREAD

**Issue by bnb** [nodejs/community-committee#318]
Since we got approval to merge the PR and kick off the CommComm Director Role, we get to begin the election process!

### Director Election Timeline
May 17 - Call for Nominations Begins
May 31 - Call for Nominations Ends
June 1 - Nominee Q&A Begins
June 8 - Nominee Q&A Ends
June 8 - Voting Opens
June 15 - Voting Ends
June 15 - Announce Winner

### What Will the CommComm Director Do?
I've already been asked this question a couple times, so I'm going to try to share the context I have. @ashleygwilliams, @williamkapke, and @MylesBorins have direct experience serving on the board for the project, so I'll also invite them to share their experiences in their unique roles.

The CommComm Director will be serving on the Node.js Board of Directors as a liaison of the Community Committee and its interests. You will be tasked with ensuring that the CommComm's perspective and interests are represented on the Board of Directors, introducing and driving forward initiatives to conclusion that capture the mission of the Node.js project and its community, and representing the Board itself in some situations.

To ensure that expectations are set appropriately, I'd like to note that the CommComm Director role will likely require a non-trivial level of time commitment, as you will need to participate in board discussions, sit in on board meetings, and have the opportunity to participate in the Joint Strategy Subcommittee.

For some more context on what the Board does, I highly suggest reading [this post on the Individual Membership Director](https://medium.com/@nodejs/nominations-open-for-node-js-foundation-individual-membership-director-4f004f5cc140) election opening up in 2016. While there is certainly some outdated information, there is also quite a bit of insight contained within.

### How Does Nomination Work?
All current **Community Committee Members** can self-nominate in this issue within the given window. Once the nomination window is over, the CommComm Chairperson will begin the Q&A process on GitHub for each nominee.

### Current Nominees:
- None!

### Final Note:
This is the CommComm's first time running an election for the Director seat. If there are any suggestions/comments on how this can be improved, please don't hesitate to leave a comment or ping me privately 👍

**Comment by bnb** [comment]
Just wanted to make an additional note:

In my own **personal experience** with the Node.js Board of Directors and the seats that the Community has owned (TSC Director, Individual Membership Directors), there is not a lot of guidance provided for the directors. We're getting better at this as a whole, but as the _first CommComm Director_ you will likely end up both setting expectations for and defining scope of the CommComm director. The entire Community Committee will be here to help you with this, and I imagine that feedback loop will be important to success.

**Comment by bnb** [comment]
Finally, because we'll be having a Q&A period for the nominees, I'd like to ask that we leave any questions for them until that time 🙏

**Comment by williamkapke** [comment]
It's helpful to take a look at:
https://github.com/nodejs/board/blob/master/Onboarding_Guide_for_Directors.md

**Comment by hackygolucky** [comment]
My story with the Node.js community started back in 2013 when I was [welcomed with open arms](https://medium.com/@hackygolucky/community-conferences-and-donuts-do-you-really-want-to-know-how-they-are-made-7b4cf8fe3c56) by Ben Acker to help organize the PDXNode meetup. We started [one of the early NodeSchools](https://medium.com/@hackygolucky/pdxnode-presents-nodeschool-a727272dd4b3) in Portland, much to our delight in the amazing connections we'd form and eventually see contributing to the Node.js project and ecosystem at large. Through NodeConfs and online work, I met and was inspired by more and more people around the world excited about the communities and ecosystem we were building up around the Node.js project. I saw firsthand how our choices effected so many people trying to work and enjoy programming in the larger open source world. I began event by event, and then through commits, to contribute to the Node.js project through the lens I saw most valuable to the next phase of work missing as an organized strategy 
in Node.js–community. 

As former Education Community Manager and current Node.js contributor, I worked to draft the first charter of the Community Committee and build consensus with the TSC members, community members, and the Board of Directors to establish an equal footing for Community as valued contributions, both code and non-code, as a top-level committee in the Node.js Foundation and project. 

I believe that the ecosystem benefits most when we are honing our communications between these separate groups who all play a vital part in the story of Node.js. I feel that with my deep understanding of the foundation and project, I will able to help it continue to progress through the role of director on the Node.js Board of Directors. In the hopes of acting in the best interest of Node.js as a trusted, high fidelity liaison to the CommComm and the incredible work of contributors we are lucky enough to share time with, I am running for the CommComm Director seat on the Node.js Foundation Board of Directors. Please accept my deepest thanks for your consideration to vote for me on the Board in the upcoming election.

**Comment by chowdhurian** [comment]
"Current Nominees:" updated.

**Comment by MylesBorins** [comment]
I believe that the nomination period has passed and @hackygolucky is the only nominee!

🎉

**Comment by bnb** [comment]
Yep, since the nomination period has passed @hackygolucky wins by default since there were no other nominees.

I’d like to congratulate her on becoming the first Community Committee Director! 🎉🎊🎉🎊🎉

**Comment by bnb** [comment]
As a point of process, I’d like to note that we’ll still do our week of Q&A with Tracy 👍

**Comment by Tiriel** [comment]
Since this issue was created as a Call For Nominations, I was wondering if we should close it.

For the time being, I'll just rename it since the agenda in the OP seem useful still. Feel free to close otherwise!

**Comment by chowdhurian** [comment]
There's a separate q/a issue open: #331

**Comment by bnb** [comment]
We can close this now - thanks @Tiriel and @chowdhurian 👍
